src/ert/_c_wrappers/enkf/ensemble_config.py

```diff
@@ -264,22 +292,70 @@ def gen_data_node(gen_data: List[str]) -> Optional[EnkfConfigNode]:
             report_steps,
         )
 
+    def addConfigNodeMeta(
+        self,
+        key: str = "",
+        forward_init: bool = False,
+        init_file: str = "",
+        input_file: str = "",
+        output_file: str = "",
+        var_type: Optional[EnkfVarType] = EnkfVarType.INVALID_VAR,
+    ):
+        metaInfo = ConfigNodeMeta(
+            key, forward_init, init_file, input_file, output_file, var_type
+        )
+        self._config_node_meta[key] = metaInfo
+
+    def _createNodeMetaInfo(
```

Comment:
I think it makes sense to use snake_case for all new functions. I don't think we have said that explicitly, but it is pep8.